src/startupscriptgenerator/src/common/fsvalidation.go

```diff
@@ -60,6 +60,29 @@ func WriteScript(filePath string, command string) {
 	ioutil.WriteFile(filePath, []byte(command), 0755)
 }
 
+// Appends command to a file
+func AppendScript(filePath string, command string) {
+
+	if command == "" {
+		return
+	}
+
+	file, err := os.OpenFile(filePath, os.O_APPEND|os.O_WRONLY, 0755)
+	if err != nil {
+		fmt.Println(err)
```

Comment:
We should provide a little more detail here for the user, something like the following:

```go
fmt.Println("Unable to read provided file to append command to. Error: " + error)
```